desktop/extensions-sdk/dev/api/reference/interfaces/Docker.md

```diff
@@ -87,7 +87,7 @@ const containers = await ddClient.docker.listContainers();
 
 `Promise`<`unknown`\>
 
-___
+---
```

Comment:
I _think_ the `desktop/extensions-sdk/` directory (or at least _parts of it_) is generated from source, so these should be fixed in the code generating the Markdown.

(and at least temporarily excluded from listing)

/cc @gtardif